docs/concepts/faq.md

```diff
@@ -26,6 +26,10 @@ The quickest way to get IPFS up and running on your machine is by installing [IP
 
 For installing and initializing IPFS from the command line, check out the [command-line quick start](../how-to/command-line-quick-start.md) guide.
 
+### Why doesn't the SHA-256 hash of the file match the IPFS CID?
```

Comment:
Heads up that this exists https://docs.ipfs.io/concepts/hashing/#content-identifiers-are-not-file-hashes. This concise answer is reasonable, but figured we might want to only answer in one place (or link the answers together)

Reply:
This FAQ was triggered by my issue https://github.com/ipfs/ipfs-docs/issues/996, where I requested this more concise answer.

Reply:
Thanks, @aschmahmann. I'm still getting to know the current state of our information architecture. Although it's fair to have an FAQ repeat info in different words, this FAQ is out of place in this high-level set of questions. Let me see if I can either reword the [hashing article](https://docs.ipfs.io/concepts/hashing/#content-identifiers-are-not-file-hashes) to make it more clear or find another place for the "concise answer."
cc: @Johnmullhausen: What do I do if I changed my mind about a PR?

Reply:
Working on rewording the Hash article. Closing this PR.

Reply:
@vmx: Here is the new PR that addresses the issue from a bird's-eye-view: https://github.com/ipfs/ipfs-docs/pull/1022